app/views/registers/show.html.haml

```diff
@@ -1,5 +1,5 @@
-- @page_title = "#{@register.register_name} register"
-- @page_description = @register.register_description
+- @page_title = "#{@register.seo_title.present? ? @register.seo_title : @register.register_name + ' register'}"
```

Comment:
I think you could express this more concisely as `[@register.seo_title, @register.register_name].find(&:present?)`